src/templates/presses.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import ReactMarkdown from "react-markdown";
+import RedLayout from '../layouts/red';
+import PrimaryPageCTA from "../components/primary-page-cta";
+import SEO from "../components/seo";
+import Hero from "../components/hero";
+
+const PressTemplate = ({ data }) => {
```

Comment:
Another nitpick, but it would be better to have the component name match the file name. So in place of `presses.js` maybe `press-release.js` makes more sense and then the component could be `PressRelease`

Reply:
Good thinking. Fixed